### DISCUSSION TITLE
Questions about dp test

### DISCUSSION THREAD

**Discussion by Lihuigao21** [deepmodeling/deepmd-kit#2525] [Q&A]
Now I get a model and then I want to test it use 'dp test'. But I find that using this test for a single point or for a md process have a great difference.
For single point, dp test usually gives a big energy error, such as 12 eV, but when I run a md, this error goes down to nearly0.05eV. For example ,I test a point that in my sample set，the dp test result:
data_e                                        pred_e
-5.511939714300000333e+02 -5.519784879380499660e+02  

nearly 0.8eV error

But I try a 10 steps md:
data_e pred_e
-5.520190768899999512e+02 -5.519784879380499660e+02
-5.520191498299999466e+02 -5.519779896038156721e+02
-5.520592667299999903e+02 -5.520154537643435333e+02
-5.521286317300000519e+02 -5.520798367537207696e+02
-5.522081949399999985e+02 -5.521515189859049997e+02
-5.522751725599999872e+02 -5.522079580493785897e+02
-5.523114247099999830e+02 -5.522317821506122755e+02
-5.523144550099999606e+02 -5.522224417781244483e+02
-5.523036728099999664e+02 -5.522019713807354719e+02
-5.523115061500000138e+02 -5.522056214987596832e+02

Energy MAE         : 6.774895e-02 eV

So I doubt that which one is  persuasive？Can dptest be used to predict one single point like this, namely read the single point optimization OUTCAR?

**Comment by njzjz** [comment]
Do you have different DFT configurations for single point calculations and MD simulations? It seems that you have two different potential energies for the same coordinates. (-5.511939714300000333e+02 vs -5.520190768899999512e+02) You could firstly check if the reference data is accurate.
